Automattic/pocket-casts-android
# Update podcasts grid design

Part of #2081 

## Description

This updates Podcasts tab grid design.
Figma Design: G60UqxPhTynbalo0Vfr1NC-fi-1255_8864

** I created a separate feature flag for these changes currently disabled in Firebase. It can be enabled in 7.64 if required.

## Testing Instructions
1. Open Podcasts tab
2. Notice the grid spacing matches the design
3. Tap more menu in the toolbar
4. Select All unfinished for Badges
5. ✅ Notice that a new unfinished badge is shown
6. Select Only latest episode
7. ✅ Notice that a new only latest badge is shown
8. Switch to list view
9. ✅ Notice that new badges(similar to iOS) are shown for the list view ** <- (Needs clarification G60UqxPhTynbalo0Vfr1NC-fi-1255_8864#793222421.  I matched it with iOS.)
10. Disable "Podcast grid view design changes" feature flag 
11. Kill and Restart the app
12. ✅ Notice that original grid design changes are shown

## Screenshots or Screencast 

![1](https://github.com/Automattic/pocket-casts-android/assets/1405144/f157f8f6-c899-4ab3-82eb-b0135af6b9b2)

![2](https://github.com/Automattic/pocket-casts-android/assets/1405144/a4bb74ac-80b4-4559-82b5-a64be8846a57)

![out-1](https://github.com/Automattic/pocket-casts-android/assets/1405144/dd794e4c-086b-41db-8ae3-907694bf97dd)

![out-2](https://github.com/Automattic/pocket-casts-android/assets/1405144/ea67a067-2c07-4cdb-8e83-55cdfd69cf71)


## Checklist
- [ ] I have considered whether it makes sense to add tests for my changes N/A
 
#### I have tested any UI changes...
